spec/system/volunteers/index_spec.rb

```diff
@@ -173,13 +191,13 @@
       expect(page.all("table#volunteers tbody tr").count).to eq 1
 
       click_on "Status"
-      find(:css, 'input[data-value="Active"]').set(false)
-
-      # when all users are hidden, the tr count will be 1 for "no results" row
-      expect(page.all("table#volunteers tbody tr").count).to eq 1
-
-      find(:css, 'input[data-value="Inactive"]').set(true)
+      find(:css, 'input[data-value="true"]').set(false)
+      expect(page).to have_text("No matching records found")
```

Comment:
We might want to make this copy more friendly- something like "No volunteers match your filters. Try removing some filters and try again!"